### ISSUE TITLE
6/8/17 Update

### ISSUE THREAD

**Issue by kylemathias** [open-learning-exchange/take-home#59]
Last Meeting: #58 
Me and @zeivhann met today in hangouts and talked about our progress on our current project https://github.com/open-learning-exchange/take-home/issues/57 , and what our next steps should be.

We currently have 2 different ways to connect to the RBpi and use SSH commands. One is using a network where both the android device and the RBpi are connected to the same network. The other way is using a USB cable between the RBpi and the Android device to establish a connection. Both ways are laid out in a guide i created this week found here: [Android SSH RBPi setup](https://docs.google.com/document/d/1_tpV8_51b0FThs_WZI4Yf1mww-Cz8ELb1e86o74oDCA/edit) 

Once connected with SSH we can then send commands over to configure the wifi thus solving one of our goals for our project #57. There is an app that can make an SSH connection and then send over stored Script commands found here: [SSH button](https://play.google.com/store/apps/details?id=com.pd7l.sshbutton&hl=en) we would like people to help test. Here are some steps to help guide you to help us test some of our proposed solutions.

1. Mount and run the custom RBpi image found [here](http://pirate.sh/op/treehouse3.img.gz)
2. Follow my guide [Android SSH RBPi setup](https://docs.google.com/document/d/1_tpV8_51b0FThs_WZI4Yf1mww-Cz8ELb1e86o74oDCA/edit) and try to connect to send some ssh commands over.

Our goal for this upcoming week is to find a way to mount the RBpi image with and Android device. There are some apps that say they can do it, but we haven't been able to get any of them to work yet. When you find a possible solution, post it [here](https://docs.google.com/document/d/1fnxF_mV_JjS52h6lKUJg-MQ0tgeGDZV1XcSc3BpLsIc/edit) so other people can try it out.

Other interns, please feel free to post additional topics i might have missed, and post a comment here that you have read this.
Next week: https://github.com/open-learning-exchange/take-home/issues/61

**Comment by timecomplexity** [comment]
I have read this.

I have tried out SSH button and it appears to work. It doesn't make windows pop up or anything on the pi itself, but it does seem to deliver commands. We can probably use this app. Or, building our own probably wouldn't be the hardest thing in the world, although maybe it would be unnecessary. I did this using USB tethering from my phone.

Unfortunately, I'm not having much luck with finding a way to flash the SD card without the help of a PC. I will keep looking.

**Comment by kylemathias** [comment]
i was able to root my old galaxy s4. I am going to try out the imaging software.

**Comment by Yurockkk** [comment]
I try to connect android device to RPi through bluetooth. I followed some instructions I found ( I need a little tweak on the python code provided [here](https://www.hackster.io/user16726/raspberry-pi-bluetooth-terminal-f7e9bd) to get it run on my RPi 3) along with the android app "[Bluetooth Terminal](https://play.google.com/store/apps/details?id=Qwerty.BluetoothTerminal&hl=en)" and now I’m able to:

1. Connect android device to RPi through bluetooth
2. Send commands from android device to RPi and the output will send back to the android through bluetooth

Demo: 

![bluetooh_control](https://user-images.githubusercontent.com/11954766/27152479-ffffc86e-511b-11e7-832c-3134c0b436f6.jpg)

Potential problem:
not sure if I can change the wifi settings through bluetooth
